src/components/App/App.tsx

```diff
@@ -0,0 +1,61 @@
+import React, { useState } from 'react';
+import Form from '../Form/Form'
+// import logo from './logo.svg';
+import './App.css';
+
+const App = () => {
+
+  const [name, setName] = useState('')
+  const [username, setUsername] = useState('')
+  const [email, setEmail] = useState('')
+  const [page, setPage] = useState(0)
+  const [make, setMake] = useState('')
+  const [model, setModel] = useState('')
+  const [year, setYear] = useState('')
+  const [origin, setOrigin] = useState('')
+  const [destination, setDestination] = useState('')
```

Comment:
I wonder if there is a way to condense this somehow? Maybe for a future refactor. Otherwise this looks great!